pkgs/development/python-modules/anthropic/default.nix

```
@@ -56,11 +64,20 @@ buildPythonPackage rec {
     respx
   ];
 
+  pytestFlagsArray = [
+    "-W ignore::DeprecationWarning"
+  ];
+
   disabledTestPaths = [
     # require network access
     "tests/api_resources"
   ];
 
+  disableTests = [
+    # require network access
+    "test_copy_build_request"
```

It doesn't seem like this has anything to do with network access.

```suggestion
    "test_copy_build_request" # upstream needs to fix a memory leak
```